source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -292,7 +292,7 @@ namespace AdaptiveSharedNamespace
         unsigned int maxActions = 5;
         Spacing spacing = Spacing::Default;
         IconPlacement iconPlacement = IconPlacement::AboveTitle;
-        unsigned int iconSize = 30;
+        unsigned int iconSize = 16;
```

Comment:
@golddove 
How would we know if the iconSize is not set? Shouldn't we change it to optional to tell if it has a value?

Reply:
The plan of record was to always respect iconSize.